en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/extensions/request-body-passing.md

```diff
@@ -0,0 +1,26 @@
+You can enable request body passing in Choreo Connect using below configurations.
```

Comment:
Explain two possible ways to enable / disable feature

Reply:
Fixed with: https://github.com/wso2/docs-apim/pull/5242/commits/f44c1a16d282eaa7ce961f20242989baed0f197c